### PR TITLE
[WIP] Gives EVERY species it own set of background and languages.

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request

- [ ] Creates several languages and such.
- [ ] Ports or Creates cultural backgrounds and adapted them to our codebase.
- [ ] Fix skrell having liver failure just to sip one alcohol drink...[oopies]

My plan is to give more cultural background and languages to people and such.
For example, giving xeno-hybrid its own language instead of xeno-common like yogstation about Polysmorph

## Why It's Good For The Game
Customization is good!

## Changelog
Don't bother with this for now
/:cl:

### CODE REVIEW

**Comment by DiscountDirtyDan** [comment]
What human languages will be added?

**Comment by Guidesu** [comment]
> You currently have a negative Fix/Feature pull request delta of -18. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

Yee thank you

**Comment by Guidesu** [comment]
> What human languages will be added?

Dunno go ahead and give ideas

**Comment by desertmote** [comment]
Please DM me so we can talk at 'ghost sheep#5498', I'm currently on species design and expression. It would be better to discuss with me so we don't end up with conflicting code, as we have **very** similar ideas in our grand plans.

**Comment by Azarak** [comment]
First and foremost, atomize

**Comment by Guidesu** [comment]
> First and foremost, atomize


image

**Comment by Draganfrukts** [comment]
> 
> 
> > What human languages will be added?
> 
> Dunno go ahead and give ideas

Oldbase had Solar Common. @Guidesu

**Comment by NullDagaf** [comment]
> ### Conflicting files
> tgstation.dme

how

**Comment by Guidesu** [comment]
> > ### Conflicting files
> > tgstation.dme
> 
>how

I ask this every day...
How...
How is life..
How i can..
Live...?
How i make an PR?

**Comment by Guidesu** [comment]
image

**Comment by NullDagaf** [comment]
Imagine having a master branch.

_This post was made by the "branch out of origin" gang._

**Comment by GoldenAlpharex** [comment]
Does this mean I won't have to add Nekomimetic to Felinids myself? If so, cheers.

**Comment by NullDagaf** [comment]
> Does this mean I won't have to add Nekomimetic to Felinids myself? If so, cheers.

Nekomimetic is a-
Wait, I can't curse on GitHub. Uh...
Oh, I know.

Nekomimetic is a r####### thing that felinids should not f###ing have. Just because you stitched cat ears to your already dented head does not make it okay to have an ENTIRE NEW F###ING LANGUAGE **JUST FOR THESE CAT F####!!**

If you wanna have your "cat language", at LEAST, AT THE VERY LEAST, make it an actual lore-friendly language than, and I quote, "an incomprehensible mess of broken Japanese". Like, i dunno, something gene-modding fanatics came up with to further cement their differences/superiority/whatever over the non-modding masses? That sounds MUCH better than just "haha funny cat language go owo"



~~On that note, give ashies their own language, and make them not speak draconic. Unless you specifically imply and support that most lizardpeople are civilised ashwalkers, which I'm quite sure most people here would not be a fan of.~~

**Comment by Guidesu** [comment]
> > Does this mean I won't have to add Nekomimetic to Felinids myself? If so, cheers.
> 
> Nekomimetic is a-
> Wait, I can't curse on GitHub. Uh...
> Oh, I know.
> 
> Nekomimetic is a r####### thing that felinids should not f###ing have. Just because you stitched cat ears to your already dented head does not make it okay to have an ENTIRE NEW F###ING LANGUAGE **JUST FOR THESE CAT F####!!**
> 
> If you wanna have your "cat language", at LEAST, AT THE VERY LEAST, make it an actual lore-friendly language than, and I quote, "an incomprehensible mess of broken Japanese". Like, i dunno, something gene-modding fanatics came up with to further cement their differences/superiority/whatever over the non-modding masses? That sounds MUCH better than just "haha funny cat language go owo"
> 
> ~On that note, give ashies their own language, and make them not speak draconic. Unless you specifically imply and support that most lizardpeople are civilised ashwalkers, which I'm quite sure most people here would not be a fan of.~

can do ashwalkers thing

**Comment by Guidesu** [comment]
image
of course if dream maker helps me ;)

**Comment by FlamingLily** [comment]
> 
> 
> > Does this mean I won't have to add Nekomimetic to Felinids myself? If so, cheers.
> 
> Nekomimetic is a-
> Wait, I can't curse on GitHub. Uh...
> Oh, I know.
> 
> Nekomimetic is a r####### thing that felinids should not f###ing have. Just because you stitched cat ears to your already dented head does not make it okay to have an ENTIRE NEW F###ING LANGUAGE **JUST FOR THESE CAT F####!!**
> 
> If you wanna have your "cat language", at LEAST, AT THE VERY LEAST, make it an actual lore-friendly language than, and I quote, "an incomprehensible mess of broken Japanese". Like, i dunno, something gene-modding fanatics came up with to further cement their differences/superiority/whatever over the non-modding masses? That sounds MUCH better than just "haha funny cat language go owo"
> 
> ~On that note, give ashies their own language, and make them not speak draconic. Unless you specifically imply and support that most lizardpeople are civilised ashwalkers, which I'm quite sure most people here would not be a fan of.~

Pretty sure in lore (or at least, in TG lore, we don't have any) ashwalkers were a result of a freak bluespace teleportation accident that sent a colony of lizardpeople to lavaland, hence why they know the same language. I wouldn't mind the separation, but it could be fun if like, lizardpeople understood half the ashies language, not all of it, but like some of it. If that's even possible.

Also uhhhhh leave ashwalkers knowing draconic, make lizardpeople know Sinta'Unathi (and just like rename lizardpeople to Unathi properly (and have a "anthromorphic reptile" species) cause they're the same) cause it's better than "draconic" (WHICH THE DAMN DRAGONS DON'T SPEAK)

**Comment by NullDagaf** [comment]
> Pretty sure in lore (or at least, in TG lore, we don't have any)

Then let's make some, and not wait for whenever the- Wait, I can't shittalk admins. Sad!

> ashwalkers were a result of a freak bluespace teleportation accident that sent a colony of lizardpeople to lavaland, hence why they know the same language.

I have never heard that being a thing. In my mind they're just one of lavaland's natives, and that's it. ~~Also screw whoever thinks they're bioengineered superweapons made by the tendril, IF THEY WERE YOU WOULDN'T HAVE ANY OTHER SPECIES ON THERE NOW WOULD YOU~~

>I wouldn't mind the separation, but it could be fun if like, lizardpeople understood half the ashies language, not all of it, but like some of it. If that's even possible.

That could probably be coded in, although it'd require re-doing how languages work.
Also, come on. Just because you have scales does not mean you can understand everyone else who has them too.

> Also uhhhhh leave ashwalkers knowing draconic, make lizardpeople know Sinta'Unathi (and just like rename lizardpeople to Unathi properly (and have a "anthromorphic reptile" species) cause they're the same) cause it's better than "draconic" (WHICH THE DAMN DRAGONS DON'T SPEAK)

Watch people bitch, as their favorite kobold waifu is now an "Unathi", **even if they could remake them with anthro reptile.**
Also c'mon. Make races of your own instead of ripping from.. wherever Unathi are from.

File: modular_skyrat/modules/customization/modules/language/xeno.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/datum/language/xenohybrid
+	name = "Xenomorphic Common"
+	desc = "An uncommon tongue derived from extreme long contact with xenomorphs, a growling, hissing language full of glottal stops. This language however it's they own type of language which not even aggressive xenomorph can understand."
```
Comment (NullDagaf) [inline modular_skyrat/modules/customization/modules/language/xeno.dm:3]: Grammar.

**Comment by H0gberg** [comment]
I've got some ideas for human languages, lore can be added to em if ya want

- Neo-Latin which is spoken by colonized planets on the frontier
- Post Glagolitic which is based on the Cyrillic alphabet and is basically space russian
- Sign Language
- Neo-Aramaic which is a future version of Arabic

**Comment by Guidesu** [comment]
> I've got some ideas for human languages, lore can be added to em if ya want
> 
> * Neo-Latin which is spoken by colonized planets on the frontier
> * Post Glagolitic which is based on the Cyrillic alphabet and is basically space russian
> * Sign Language
> * Neo-Aramaic which is a future version of Arabic

Good ideas in general i made a lot of progress but sadly my github won't send it

**Comment by hugmeorelse** [comment]
There's no point to this currently. I am closing this. (This should be atomized anyways)

We'll talk more one on one, so you can have official backing on this. :)

**Comment by Guidesu** [comment]
what

**Comment by Guidesu** [comment]
Thank you for closing the PR which dear god i had an fuckton to shit to send

**Comment by Guidesu** [comment]
👍

**Comment by hugmeorelse** [comment]
> 👍

🥇

**Pull request closed without merging**